Build fails with "lazy" + "ipc" features

#### Are you using Python or Rust?

Rust

#### Which feature gates did you use?

lazy and ipc seem to be the relevant ones

The full set this triggered with was:

    polars = { version = "0.18", features = ["lazy", "decompress", "dtype-date", "strings", "mode", "diagonal_concat", "ipc"] }

#### What version of polars are you using?

v0.18.0

#### What operating system are you using polars on?

Linux

#### Describe your bug.

If running with `lazy` and `ipc` the library fails to build:

```
error[E0599]: no function or associated item named `new` found for struct `polars_io::ipc::IpcReader` in the current scope
   --> /home/dbr/.cargo/registry/src/github.com-1ecc6299db9ec823/polars-lazy-0.18.0/src/logical_plan/mod.rs:846:42
    |
846 |         let schema = Arc::new(IpcReader::new(file).schema()?);
    |                                          ^^^ function or associated item not found in `polars_io::ipc::IpcReader<_>`
    |
    = help: items from traits can only be used if the trait is in scope
    = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
            `use polars_io::SerReader;`
```

https://github.com/pola-rs/polars/blame/1606cbc4a98988a4fded0d73f661ee89d1e58af7/polars/polars-lazy/src/logical_plan/mod.rs#L848

As writing this I noticed including the `parquet` feature works around the problem due to this line:
https://github.com/pola-rs/polars/blame/1606cbc4a98988a4fded0d73f661ee89d1e58af7/polars/polars-lazy/src/logical_plan/mod.rs#L19-L20

#### What are the steps to reproduce the behavior?

`cargo new` then

```
[package]
name = "polarsipcbug"
version = "0.1.0"
edition = "2018"

[dependencies]
polars = { version = "0.18", features = ["lazy", "ipc"]}
```

Then `cargo build`

--- a/polars/polars-lazy/src/logical_plan/mod.rs
+++ b/polars/polars-lazy/src/logical_plan/mod.rs
@@ -17,7 +17,7 @@ use polars_core::prelude::*;
 #[cfg(feature = "csv-file")]
 use polars_io::csv_core::utils::infer_file_schema;
 #[cfg(feature = "parquet")]
-use polars_io::{parquet::ParquetReader, SerReader};
+use polars_io::parquet::ParquetReader;
 
 use crate::logical_plan::LogicalPlan::DataFrameScan;
 use crate::utils::{
@@ -824,6 +824,8 @@ impl LogicalPlanBuilder {
         n_rows: Option<usize>,
         cache: bool,
     ) -> Result<Self> {
+        use polars_io::SerReader as _;
+
         let path = path.into();
         let file = std::fs::File::open(&path)?;
         let schema = Arc::new(ParquetReader::new(file).schema()?);
@@ -843,6 +845,8 @@ impl LogicalPlanBuilder {
     #[cfg(feature = "ipc")]
     #[cfg_attr(docsrs, doc(cfg(feature = "ipc")))]
     pub fn scan_ipc<P: Into<PathBuf>>(path: P, options: IpcOptions) -> Result<Self> {
+        use polars_io::SerReader as _;
+
         let path = path.into();
         let file = std::fs::File::open(&path)?;
         let schema = Arc::new(IpcReader::new(file).schema()?);

